Makefile

```diff
@@ -1,7 +1,9 @@
-.PHONY: all install install-dependencies install-tools test test-verbose
+.PHONY: all install install-dependencies install-tools test test-full test-verbose
```

Comment:
True, still atm they do the same. Please correct on what I'm currently obviously not seeing.